tests/e2e/api/account.test.ts

```diff
@@ -137,6 +137,71 @@ describe("account api", () => {
       expect(accountTransactions[0]).toStrictEqual(txn);
     });
 
+    test("it fetches all account transactions", async () => {
+      const config = new AptosConfig({ network: Network.LOCAL });
+      const aptos = new Aptos(config);
+      const alice = Account.generate();
+      await aptos.fundAccount({
+        accountAddress: alice.accountAddress,
+        amount: FUND_AMOUNT,
+      });
+      const bob = Account.generate();
+      await aptos.fundAccount({
+        accountAddress: bob.accountAddress,
+        amount: FUND_AMOUNT,
+      });
+      const rawTxn1 = await aptos.transaction.build.simple({
+        sender: alice.accountAddress,
+        data: {
+          function: "0x1::aptos_account::transfer",
+          functionArguments: [bob.accountAddress, new U64(10)],
+        },
+      });
+      const authenticator1 = aptos.transaction.sign({
+        signer: alice,
+        transaction: rawTxn1,
+      });
+      const response1 = await aptos.transaction.submit.simple({
+        transaction: rawTxn1,
+        senderAuthenticator: authenticator1,
+      });
+      const txn1 = await aptos.waitForTransaction({ transactionHash: response1.hash });
+      const rawTxn2 = await aptos.transaction.build.simple({
+        sender: bob.accountAddress,
+        data: {
+          function: "0x1::aptos_account::transfer",
+          functionArguments: [alice.accountAddress, new U64(10)],
+        },
+      });
+      const authenticator2 = aptos.transaction.sign({
+        signer: bob,
+        transaction: rawTxn2,
+      });
+      const response2 = await aptos.transaction.submit.simple({
+        transaction: rawTxn2,
+        senderAuthenticator: authenticator2,
+      });
+      const txn2 = await aptos.waitForTransaction({ transactionHash: response2.hash });
+
+      // sleep for 500 ms to ensure the txns are in the ledger
+      await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
without the sleep tx2 wouldn't be included, pretty weird considering i already waitForTransaction on tx 2